### PR TITLE
Remove play button/Start timer only when a game is started/Refactor g…

### PR DESCRIPTION
**Pull request by joshpham97**
…ame difficulty selection

### CODE REVIEW

**Comment by joshpham97** [comment]
I will remove one of them later, since we do not need both.

**Comment by binarytrails** [comment]
> I will remove one of them later, since we do not need both.

@joshpham97 got it! perhaps, a TODO  comment would be useful cause I couldn't understand

**Comment by joshpham97** [comment]
Oh my bad I forgot to put it

**Comment by binarytrails** [comment]
@joshpham97 
to merge with ``` git rebase -i master```, I use http://meldmerge.org/ you can do ```git mergetool --tool meld``` and keep going with ```git merge --continue``` until it is all good and then ```git push -f```

File: diaries/diary_TiffanyAH KING.tex [hunk 1]
```diff
@@ -172,7 +172,46 @@ \section{Iteration 1}
 
 
 
-%\section{Iteration 2}
+\section{Iteration 2}
```
Comment (binarytrails) [inline diaries/diary_TiffanyAH KING.tex:175]: that is strange it is already on master see: https://github.com/binarytrails/kakuro/commit/a26843f41e33c9e99ada0b5c32e45b9dff65bbed

File: tests/kakuro/TestPlayerDao.java [hunk 2]
```diff
@@ -1,47 +1,40 @@
...
 
 import org.junit.Test;
 
 public class TestPlayerDao {
-
-	
 	@Test
-	public void testRegisterAlreadyRegistered() {
-		
+	public void testRegisterAlreadyRegistered(){
```
Comment (binarytrails) [inline tests/kakuro/TestPlayerDao.java:18]: same for tests already on master, there is something odd here

**Review by binarytrails — CHANGES_REQUESTED** [review]

**Comment by binarytrails** [comment]
in any cases the tests gotta pass ! aim for that

**Pull request closed without merging**